### ISSUE TITLE
Interactive Maps by OpenStreetMap are not visible

### ISSUE THREAD

**Issue by yee-bit** [hirschan/Dark-Mode-Wikipedia#54]
Hello again, thank you very much for fixing the last issue that I brought up! Made the Wikipedia reading experience much more enjoyable.

I have found that OpenStreetMap interactive maps are actually completely blacked out. I don't know if this is fixable, but it would be pretty cool if you could have a look. Thanks in advance!

![Screenshot 2024-05-25 231843](https://github.com/hirschan/Dark-Mode-Wikipedia/assets/81834767/45a7fccc-590e-43fb-aa8e-7eca9e6fecf8)

That is what the map looks like by default, without the extension enabled

![Screenshot 2024-05-25 231827](https://github.com/hirschan/Dark-Mode-Wikipedia/assets/81834767/5ff89db9-4926-4769-b520-478663f42136)

This is the map with the extension enabled.

Here is the article on US Highway Route 1 that I used for these screenshots for your reference:

https://en.wikipedia.org/wiki/U.S._Route_1

**Comment by hirschan** [comment]
Good catch! Thanks, appreciate it 🙏 Will have a look the coming days :)

**Comment by hirschan** [comment]
I've spent some time and I cannot figure out how to exclude the OpenStreetMap selector from the dark mode. Tried to disable the specific URL but does not work, unfortunately. Will need to come back to this later, or hopefully someone can help out with this one 🙏
